lib/helpers.js

```diff
@@ -68,7 +68,11 @@ const getValidDiscountRules = (discountRules, params, items) => {
             if (rule.discount.type === 'percentage') {
               value *= rule.discount.value / 100
             } else {
-              value += rule.discount.value
+              if (rule.discount_kit_subtotal) {
+                value = rule.discount.value
+              } else {
+                value += rule.discount.value
+              }
```

Comment:
```suggestion
              if (rule.discount_kit_subtotal) {
                value = rule.discount.value
              } else {
                value = Math.min(value, rule.discount.value)
              }
```

Pelo que vi aí vai entrar nesse else apenas com `discount_lowest_price`, nesse caso acho que somar o desconto fixo (com o valor do menor produto) também não faria sentido, certo?

Reply:
> Na real acho que o desconto fixo pra `discount_lowest_price` não faria sentido de nenhuma forma, mas somar o desconto fixo parece pior 🙃